test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -0,0 +1,250 @@
+---------------------------------------------------------------------------------------------------
+-- OnRCStatus common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
```

Comment:
@GetmanetsIrina ```ValidateSchema``` needs to be removed since this is was implemented for the old approach with plug-in architecture.

Reply:
Locally changed.